Misc/NEWS.d/next/Library/2024-02-07-16-14-23.gh-issue-115136.DuI2LN.rst

```diff
@@ -0,0 +1 @@
+Check return value of PyMem_Malloc() before passing to memset()
```

Comment:
We can use RST markups here.

```suggestion
Check return value of :c:func:`PyMem_Malloc` before passing to :c:func:`memset`.
```

Reply:
Thank you for review, updated entry with your suggestion

Reply:
Sorry, I updated the comment after your latest commits. I think we should add a description of which part we add the check to.

Reply:
I think there is no need in `NEWS` entry. Otherwise LGTM.

Reply:
I agree with @Eclips4, The NEWS entry should contain information understendable and useful for common Python user. The common Python user has no idea what is `Modules/getpath.c` and how this change affects their code. Should they worry? What code they should avoid? What workaround they should use and what workaround is no longer needed?

This bug has no workaround, but it can happen only in very rare circumstances, and may be shadowed by other bugs. So the common user should not worry about it, not more than about other not yet found bugs or about the OOM killer killing their program.